presto-main/src/test/java/io/prestosql/sql/planner/TestLiteralEncoder.java

```diff
@@ -79,12 +83,22 @@
     @Test
     public void testEncode()
     {
-        assertEncode(utf8Slice("hello"), VARBINARY, literalVarbinary("hello".getBytes(UTF_8)));
+        assertEncodeCaseInsensitively(utf8Slice("hello"), VARBINARY, literalVarbinary("hello".getBytes(UTF_8)));
```

Comment:
Why case insensitive? What does that mean for a VARBINARY?

Reply:
the function name here is a different case than expected
https://github.com/prestosql/presto/blob/45c96e71f6fb88d1ee4fc2a930e74d88328f2151/presto-main/src/test/java/io/prestosql/sql/planner/TestLiteralEncoder.java#L111

i guess @dain will know better

Reply:
@martint while i think it's valuable thing to fix, i don't want the PR to get blocked on it. That's why i kept current method as deprecated.